wit-manifest.json

```diff
@@ -5,7 +5,7 @@
         "source": "git@github.com:sifive/freedom-metal.git"
     },
     {
-        "commit": "89e427e15a403add4eb837f1466f0be9078049f0",
+        "commit": "379db09a9f6cea593357e966788791333ead53c5",
```

Comment:
Just to be clear, this commit hash points at an unmerged branch. Once approved I can roll up a series of PRs, bump this hash, and then this PR can be merged.